doc/source/whatsnew/v0.17.0.txt

```diff
@@ -110,6 +110,7 @@ Other enhancements
 
 - ``pd.merge`` will now allow duplicate column names if they are not merged upon (:issue:`10639`).
 
+- ``msgpack`` submodule updated to 0.4.6 (:issue:`10581`)
 .. _whatsnew_0170.api:
```

Comment:
As far as `pandas` related usage is concerned, yes; but if you use the `pandas.msgpack` for unpacking other things then there is a default argument change in `unpack`, `unpackb` and `Unpacker` (`use_list=False` instead of `True`).  Should I mention this somewhere?